src/Tribe/Shortcode/Shortcode_Abstract.php

```diff
@@ -61,7 +70,8 @@ abstract class Shortcode_Abstract implements Shortcode_Interface {
 	 * {@inheritDoc}
 	 */
 	public function setup( $arguments, $content ) {
-		$this->arguments = $this->parse_arguments( $arguments );
+		$this->arguments = Arr::parse_associative_array_alias( (array) $arguments, (array) $this->aliased_arguments );
```

Comment:
Would love to see us move this call to the `parse_arguments`.

Reply:
great idea! done.